src/app/dataset/depublication/depublication.component.html

```diff
@@ -9,6 +9,9 @@
     <li *ngIf="isSaving">
       <span class="svg-icon-spin showing"></span>
     </li>
+    <li>
+      <a (click)="onDepublishDataset()">{{ 'depublishDataset' | translate }}</a>
+    </li>
```

Comment:
I think now it is just one button, right (not a drop-down)? Later we will add drop-down options to it (in a future ticket). Just so that you're aware.

Reply:
Yep.  I know.